app/services/poi_importer.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+class PoiImporter
+  def import_from_csv(path_to_csv, kind)
+    rows = []
+    CSV.foreach(path_to_csv) do |row|
+      longitude = row[0]
+      latitude = row[1]
+      unless find_poi(longitude, latitude, kind)
+        rows.push(longitude: longitude, latitude: latitude, kind: kind) unless longitude.nil? || latitude.nil?
```

Comment:
```ruby
 unless longitude.nil? || latitude.nil? 
```
This would work only if both parameters absent. If one of them isn't nil this wouldn't be correct. false || true -> true

I think better is use a much simpler condition.
``` ruby
if longitude.present? && latitude.present?
```

I wonder a bit why do you check this in this place...